src/common/CMakeLists.txt

```diff
@@ -24,5 +24,5 @@ include(numbers/CMakeLists.txt)
 install(FILES
         StringMap.h Timer.h inttypes.h IColor.h
         TreeOps.h FlaPartitionMap.h PartitionInfo.h Partitions.h ApiException.h TypeUtils.h
-        NatSet.h ScopedVector.h TermNames.h
+        NatSet.h ScopedVector.h TermNames.h NonLinException.h
```

Comment:
Why do we need to install it?

Reply:
We would still need it in LASolver though
Or do you mean to we'll move it purely to LASolver then?

Reply:
Added it to the issue #815

Reply:
I meant, only to remove it from this place in CMakeLists.txt. Not the actual file from the `common` directory.

Reply:
Updated the PR